lib/KBreadcrumbs.vue

```diff
@@ -266,8 +270,13 @@
   }
 
   .breadcrumbs-dropdown {
+    @extend %dropshadow-16dp;
+
+    position: absolute;
+    z-index: 100;
```

Comment:
Generally we try to match dropshadow and z-index together based on default material elevations:

https://material.io/design/environment/elevation.html#default-elevations

In theory then this would be `dropshadow-8dp` and `z-index: 8`